xcode/Subconscious/Shared/Services/Noosphere/Sphere.swift

```diff
@@ -10,14 +10,16 @@ import Combine
 import SwiftNoosphere
 import os
 
+public typealias Cid = String
```

Comment:
Introduce typealias for Cid strings. This will make it easier if we want to add a wrapping type in future (ala Did or Slug).